tests/scripts/create-test-cluster.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
What about naming this `dev-cluster.sh`? I imagine it can be expanded for many conveniences for development.